sys/linux/dev_loop.txt

```diff
@@ -28,6 +29,13 @@ ioctl$LOOP_CTL_REMOVE(fd fd_loop_ctrl, cmd const[LOOP_CTL_REMOVE], num fd_loop_n
 lo_encrypt_type = LO_CRYPT_NONE, LO_CRYPT_XOR, LO_CRYPT_DES, LO_CRYPT_FISH2, LO_CRYPT_BLOW, LO_CRYPT_CAST128, LO_CRYPT_IDEA, LO_CRYPT_DUMMY, LO_CRYPT_SKIPJACK, LO_CRYPT_CRYPTOAPI
 lo_flags = LO_FLAGS_READ_ONLY, LO_FLAGS_AUTOCLEAR, LO_FLAGS_PARTSCAN, LO_FLAGS_DIRECT_IO
 
+loop_config {
+	lo_fd		int32
```

Comment:
I see in the kernel code this field is called `fd` without `lo_` prefix:
https://elixir.bootlin.com/linux/v5.18-rc5/source/include/uapi/linux/loop.h#L83
Please use the actual kernel names for fields. It makes it easier to search for them, read for kernel developers, do static checking, etc.

Reply:
This field needs to have `fd` type rather than just `int32`.